setup-dynamodb-local.mjs

```diff
@@ -0,0 +1,29 @@
+import { mkdir } from 'fs/promises'
+import { existsSync } from 'fs'
+import { execSync } from 'child_process'
+
+const dynamodbDir = 'dynamodb_local_latest'
+const dynamodbArchiveURL =
+  'https://s3.us-west-2.amazonaws.com/dynamodb-local/dynamodb_local_latest.tar.gz'
+
+// Check if the dynamodbDir directory exists
+async function runSetup() {
+  if (!existsSync(dynamodbDir)) {
```

Comment:
Since this is an async function, we might as well use the promise-returning version of this function.

Reply:
From what I saw, there is no async version of `existsSync`, is there another function that does the same, or will this just require a Promise wrapper?